samples/SocketsSample/EndPoints/MessagesEndPoint.cs

```diff
@@ -64,10 +65,11 @@ private Task Broadcast(string text)
         private Task Broadcast(byte[] payload)
         {
             var tasks = new List<Task>(Connections.Count);
+            async Task<FlushResult> ToTask(PipeAwaiter<FlushResult> awaiter) => await awaiter;
 
             foreach (var c in Connections)
             {
-                tasks.Add(c.Transport.Output.WriteAsync(payload));
+                tasks.Add(ToTask(c.Transport.Output.WriteAsync(payload)));
```

Comment:
This is a little weird. Why exactly is this happening

Reply:
WriteAsync returns custom Awaiter object, not Task that the list requires.